src/core/default-variable-storage.js

```diff
@@ -9,9 +9,10 @@ class DefaultVariableStorage {
     this.data[name] = value;
   }
 
+  // Called when a variable is being evaluated.
   get(name) {
     return this.data[name];
   }
 }
 
-module.exports = DefaultVariableStorage;
```

Comment:
Moving to es modules makes it easier for people who want to do their own compiling.